### PR TITLE
Release Eventing Kogito 1.2.0 on OLM

### PR DESCRIPTION
**Pull request by ricardozanini**
Prepare manifests to release 1.2.0 on OLM

### CODE REVIEW

**Comment by ricardozanini** [comment]
Related PRs:

- https://github.com/redhat-openshift-ecosystem/community-operators-prod/pull/733
- https://github.com/k8s-operatorhub/community-operators/pull/682

File: config/olm/manifests/bases/eventing-kogito.clusterserviceversion.yaml [hunk 1]
```diff
@@ -75,4 +75,5 @@ spec:
   minKubeVersion: 1.20.0
   provider:
     name: Red Hat
+  replaces: eventing-kogito.v1.1.0
   version: 1.1.0
```
Comment (sutaakar) [inline config/olm/manifests/bases/eventing-kogito.clusterserviceversion.yaml:79]: This version should be raised too

Comment (ricardozanini) [reply config/olm/manifests/bases/eventing-kogito.clusterserviceversion.yaml:79]: This is just the placeholder, the `operator-sdk` command replaces it for the right version: https://github.com/knative-sandbox/eventing-kogito/blob/main/Makefile#L188

Comment (sutaakar) [reply config/olm/manifests/bases/eventing-kogito.clusterserviceversion.yaml:79]: ok, good

**Pull request merged**